_includes/post-summary.html

```diff
@@ -0,0 +1,13 @@
+<div class="post-summary {{ include.featured_class }}">
+
+    <a href="{{ post.url }}">
+        <h2>{{ post.title }}</h2>
```

Comment:
in the preview, it feels a little odd not to have the dates of posts anywhere; is it feasible to add them?